### PR TITLE
Add testcase for influxDB sink and source/kubelet

### PR DESCRIPTION
**Pull request by huangyuqi**
Add testcase for influxDB sink and source/kubelet 
fixed https://github.com/kubernetes/heapster/issues/827

### CODE REVIEW

**Comment by huangyuqi** [comment]
Hi, @mwielgus Please help to check, thanks.

File: metrics/sinks/influxdb/influxdb_test.go [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+type fakeInfluxDBClient struct {
+	pnts []pointSavedToInfluxdb
+}
+
+type fakeInfluxDBSink struct {
+	core.DataSink
+	fakeInfluxDbClient *fakeInfluxDBClient
+}
+
+func NewFakeInfluxDBClient() *fakeInfluxDBClient {
```
Comment (mwielgus) [inline metrics/sinks/influxdb/influxdb_test.go]: Can we have this in common//influxdb and reuse in metrics/ and events/ ?

**Comment by mwielgus** [comment]
Looks good, just one minor comment.

**Comment by huangyuqi** [comment]
yes, i will move these together

**Comment by mwielgus** [comment]
LGTM, thank you for the PR.

**Pull request merged**